src/Http/Controllers/JsonApiController.php

```diff
@@ -23,6 +23,8 @@
 use CloudCreativity\LaravelJsonApi\Contracts\Store\StoreInterface;
```

Comment:
At the moment this PR only supports async processes being returned by the adapter. However, it would be appropriate for them to be dispatched from controller hooks instead. Especially as the controller hooks are the documented location for dispatching jobs.

Need to amend the controller to handle an async process being returned by the hooks.